pxr/imaging/hdMtlx/hdMtlx.cpp

```diff
@@ -562,6 +562,12 @@ HdMtlxCreateMtlxDocumentFromHdMaterialNetworkInterface(
 
     // Initialize a MaterialX Document
     mx::DocumentPtr mxDoc = mx::createDocument();
+
+    // Need to figure out how to read the actual value from the netInterface
```

Comment:
A thought that @spiffmon had:

1.) Rename the property info:mtlx:version or something (such that it starts with "info:").
2.) Update materialAdapter.cpp (a) and dataSourceMaterial.cpp (b) to grab all attributes with the "info:" namespace prefix, and insert them into a new VtDictionary "info" on HdMaterialNetwork (a)/a new HdContainerDataSource named "info" on the material data source (b).
3.) Update dataSourceLegacyPrim.cpp and sceneIndexAdapterSceneDelegate.cpp to translate from the VtDictionary to the ContainerDataSource and back.
4.) Finally grab the version string off the material network here.

How does that feel, design wise/difficulty wise?